INSTALL.md

```diff
@@ -415,6 +434,28 @@ then this flag is optional and defaults to `ZLIB1` if not provided.
 This flag is optional and if not provided then `GNV$LIBZSHR`, `GNV$LIBZSHR32`
 or `GNV$LIBZSHR64` is used by default depending on the pointer size chosen.
 
+### with-zstd-include
+
+    --with-zstd-include=DIR
+
+The directory for the location of the zstd include file.  This option is only
+necessary if [zstd](#zstd) is used and the include file is not
+already on the system include path.
+
+### with-zstd-lib
+
+    --with-zstd-lib=LIB
+
+**On Unix**: this is the directory containing the zstd library.
+If not provided the system library path will be used.
+
+**On Windows:** this is the filename of the zstd library (with or
+without a path).  This flag must be provided if the
+[zstd-dynamic](#zstd-dynamic) option is not also used. If `zstd-dynamic` is used
+then this flag is optional and defaults to `LIBZSTD` if not provided.
+
+**On VMS:** support TBD.
```

Comment:
We shouldn't have a PR with "TBD" in it like this.

Reply:
Well, we shouldn't *merge* PRs with "TBD".  But we shouldn't expect generic contributors to know anything about VMS, either, and if the contributor is going to ask for help figuring out what or whether to do anything for VMS as part of the PR, does it matter if the string "TBD" is used to effectuate that or not?

Reply:
This should really be "No support"; I'm not a VMS developer. I know that @levitte has been the driver for VMS support, I would accept any help from VMS support.

Reply:
I would delete 457 and let interested parties add VMS support afterwards, if they care to do so.

Reply:
> I would delete 457 and let interested parties add VMS support afterwards, if they care to do so.

This seems like a reasonable way to handle it

Reply:
Deleted.